### PR TITLE
fix: expanded formula editor code suggestions not working

### LINKED ISSUE #1503: Expanded formula editor code suggestions not working
Code suggestions, highlighting, etc in expanded editor not working 

CleanShot 2024-07-02 at 07 36 44

### PR DESCRIPTION
**Pull request by AyushAgrawal-A2**
closes #1503

### CODE REVIEW

**Comment by davidfig** [comment]
Please resolve conflicts

**Comment by AyushAgrawal-A2** [comment]
> Please resolve conflicts

@davidfig 
done

**Comment by davidkircos** [comment]
@AyushAgrawal-A2 formulas syntax highlighting and cell highlighting also don't work in the expanded editor. Could you also fix those issues in this PR?

**Comment by AyushAgrawal-A2** [comment]
> @AyushAgrawal-A2 formulas syntax highlighting and cell highlighting also don't work in the expanded editor. Could you also fix those issues in this PR?

@davidfig 
done

File: quadratic-rust-shared/src/sql/mysql_connection.rs [hunk 1]
```diff
@@ -264,7 +264,7 @@ mod tests {
         let columns = row.columns();
         let to_arrow = |index: usize| MySqlConnection::to_arrow(row, &columns[index], index);
 
-        assert_eq!(over_the_limit, false);
+        assert!(!over_the_limit);
```
Comment (davidkircos) [inline quadratic-rust-shared/src/sql/mysql_connection.rs:267]: should this be in this branch?

File: quadratic-client/src/dashboard/components/CustomRadixIcons.tsx [hunk 2]
```diff
@@ -37,14 +37,14 @@ export const SharedWithMeIcon = React.forwardRef<SVGSVGElement, IconProps>(
...
+          fillRule="evenodd"
+          clipRule="evenodd"
           d="M3.5 5.00006C3.22386 5.00006 3 5.22392 3 5.50006V11.5001C3 11.7762 3.22386 12.0001 3.5 12.0001H11.5C11.7761 12.0001 12 11.7762 12 11.5001V5.50006C12 5.22392 11.7761 5.00006 11.5 5.00006H10.25C9.97386 5.00006 9.75 4.7762 9.75 4.50006C9.75 4.22392 9.97386 4.00006 10.25 4.00006H11.5C12.3284 4.00006 13 4.67163 13 5.50006V11.5001C13 12.3285 12.3284 13.0001 11.5 13.0001H3.5C2.67157 13.0001 2 12.3285 2 11.5001V5.50006C2 4.67163 2.67157 4.00006 3.5 4.00006H4.75C5.02614 4.00006 5.25 4.22392 5.25 4.50006C5.25 4.7762 5.02614 5.00006 4.75 5.00006H3.5Z"
           fill={color}
         />
         <path
-          fill-rule="evenodd"
-          clip-rule="evenodd"
+          fillRule="evenodd"
+          clipRule="evenodd"
```
Comment (davidkircos) [inline quadratic-client/src/dashboard/components/CustomRadixIcons.tsx:47]: are these intended changes?

File: docker/localstack/scripts/init-aws.sh
Comment (davidkircos) [inline docker/localstack/scripts/init-aws.sh:1]: is this an intended change?

**Review by davidkircos — CHANGES_REQUESTED** [review]

File: quadratic-rust-shared/src/sql/mysql_connection.rs (hunk 1 above)
Comment (AyushAgrawal-A2) [reply quadratic-rust-shared/src/sql/mysql_connection.rs:267]: yup `lint:clippy` gives warning for this
already merge in qa branch in PR #1450

File: quadratic-client/src/dashboard/components/CustomRadixIcons.tsx (hunk 2 above)
Comment (AyushAgrawal-A2) [reply quadratic-client/src/dashboard/components/CustomRadixIcons.tsx:47]: yup, this throws error in chrome dev console
already merge in qa branch in PR #1450

File: docker/localstack/scripts/init-aws.sh
Comment (AyushAgrawal-A2) [reply docker/localstack/scripts/init-aws.sh:1]: yup, this matches the name used in docker-compose.yml
already merge in qa branch in PR #1450

**Comment by AyushAgrawal-A2** [comment]
On merging `qa` into this branch the above diffs are gone

**Review by davidkircos — APPROVED** [review]

**Pull request merged**